src/mokernels/mokernel.jl

```diff
@@ -4,3 +4,31 @@
 Abstract type for kernels with multiple outpus.
 """
 abstract type MOKernel <: Kernel end
+
+"""
+    matrixkernel(k::MOK, x, y)
+
+Convenience function to compute the matrix kernel for two inputs `x` and `y`. The `outputsize` keyword is only required for the `IndependentMOKernel` to indicated the number of outputs. 
```

Comment:
```suggestion
Convenience function to compute the matrix kernel for two inputs `x` and `y`.
```
The `out_dim` version might still be useful as a fall-back / in tests to check correctness of `matrixkernel` implementation (maybe a `test_mo_kernel_interface`?)